src/app/screens/LNURLPay/index.tsx

```diff
@@ -275,8 +276,8 @@ function LNURLPay() {
     return (
       <Container maxWidth="sm">
         <PublisherCard
-          title={getRecipient()}
-          description={navState.origin?.description}
+          title={navState.origin?.name as string}
```

Comment:
Why do we need the `as` here? It can be undefined because i.e. via "Send" we do not have an origin, right?
Why forcing this to be a string then?
Same question for similar places in this PR.

If this is related to above...:
```ts
  if (!title) {
    title = description as string;
```
...why can't we make `title` optional and handle this in the screen here?

Just some thoughts. Happy to pair.

p.s.
The `PublisherCard` component seems to be quite crucial and used in a lot of places in different ways. Would be nice to add some unit-tests for that (later on).